dts/bindings/gpio/ite,it8xxx2-gpio.yaml

```diff
@@ -8,16 +8,17 @@ compatible: "ite,it8xxx2-gpio"
 include: [gpio-controller.yaml, base.yaml]
 
 properties:
-    port-is-output:
-      type: boolean
-      description: Indicates if the port is an output port
-
     reg:
       required: true
 
     label:
       required: true
 
+    index:
```

Comment:
Remove 17-21. Index will no longer be in the DeviceTree

Reply:
Done.